src/selenium/error.cr

```diff
@@ -1,12 +1,15 @@
-class Selenium::Error < Exception
```

Comment:
I have a field for message and Exception also has a message field which was causing problems